internal/testplanet/planet.go

```diff
@@ -483,6 +483,15 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 				MaxInlineSegmentSize: 8000,
 				Overlay:              true,
 				BwExpiration:         45,
+				RS: metainfo.RSConfig{
+					MaxBufferMem:     memory.Size(256),
+					ErasureShareSize: memory.Size(256),
+					MinThreshold:     (planet.config.StorageNodeCount * 1 / 5),
+					RepairThreshold:  (planet.config.StorageNodeCount * 2 / 5),
+					SuccessThreshold: (planet.config.StorageNodeCount * 3 / 5),
+					MaxThreshold:     (planet.config.StorageNodeCount * 4 / 5),
+					Validate:         true,
```

Comment:
If you will enable validation for all testplanet tests you will need to update lots of tests. Maybe by default disable this for testing and just add test to verify that this functionality is working ok.